src/machines/sketchSolve/sketchSolveMode.ts

```diff
@@ -75,7 +83,10 @@ export const sketchSolveMachine = setup({
       type: 'sketch solve tool changed',
       data: { tool: null },
     }),
-    'spawn tool': assign(({ event, spawn, context }) => {
+    'spawn tool': assign(({ event, spawn: _spawn, context }) => {
```

Comment:
The underscore prefix should only be used to indicate that a variable isn't used.

Reply:
Fair, change it.